setup-certs.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+CERT_CIRECTORY="certs"
```

Comment:
there's a small typo here
```suggestion
CERT_DIRECTORY="certs"
```